packages/react/src/lib/accessibility/FocusZone/AutoFocusZone.types.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+
+export interface AutoFocusZoneProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Element type the root element will use. Default is "div".
+   */
+  as?: React.ReactType
+
+  /**
+   * Indicates the selector for first focusable item.  Only applies if focusPreviouslyFocusedInnerElement == false.
```

Comment:
No needd `Only applies if focusPreviouslyFocusedInnerElement == false.`

Reply:
Changed to: `AutoFocusZone is used to focus inner element on mount (the first focusable or specified by the firstFocusableSelector prop).`